src/main/resources/swagger/listings/workspaces

```diff
@@ -198,12 +198,44 @@
 				"message": "Successful Request"
 			}, {
 				"code": 404,
-				"message": "Workspace does not exists"
+				"message": "Workspace does not exist"
 			}, {
 				"code": 500,
 				"message": "Rawls Internal Error"
 			}]
-		}]
+		}, {
+            "method": "DELETE",
+            "summary": "Delete workspace",
+            "notes": "",
+            "type": "Workspace",
+            "nickname": "list",
```

Comment:
Silly quibbles, but this method doesn't actually return a Workspace, and the nickname is wrong.